modules/src/flt_arith/build.lua

```diff
@@ -24,4 +23,18 @@ clibrary {
 	}
 }
 
+-- The test program isn't built by default.  Here is a target
+-- modules/src/flt_arith+pkg to install it as flt_test
 
+cprogram {
+	name = "test",
+	srcs = { "./test.c" },
+	deps = { "+lib" },
+}
+
+installable {
```

Comment:
We have old test programs in em_test, lang/cem/ctest, lang/m2/test, and elsewhere, but I don't know how to build or run any of them.